### PR TITLE
Responsive navbar

### PR DESCRIPTION
**Pull request by alt1nn**

### CODE REVIEW

File: header.php [hunk 1]
```diff
@@ -4,15 +4,10 @@
     <meta charset="<?php bloginfo('charset'); ?>">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <?php wp_head(); ?>
-   
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/all.min.css">
```
Comment (andiemini7) [inline header.php:7]: Remove the cdn, pull from develop and use the tailwindcss that we implemented

File: index.php [hunk 2]
```diff
@@ -1,71 +1,95 @@
...
+                <div class="border border-gray-200 p-4 rounded shadow bg-white">
                     <h2 class="text-2xl font-bold mb-2"><a href="<?php the_permalink(); ?>" class="text-blue-500"><?php the_title(); ?></a></h2>
                     <p><?php the_excerpt(); ?></p>
                 </div>
             <?php endwhile; ?>
         </div>
 
         <div class="my-4">
-            <?php 
+            <?php
```
Comment (andiemini7) [inline index.php]: Remove these paginations

File: index.php [hunk 3]
```diff
@@ -1,71 +1,95 @@
...
+
     <h1 class="text-4xl font-bold text-center my-4">Unified Search Results</h1>
 
     <form action="/" method="get" class="flex justify-center my-4">
-        <input type="text" name="s" class="border border-gray-300 p-2 rounded-l w-full max-w-md" placeholder="Search...">
+        <input type="text" name="s" class="border border-gray-300 p-2 rounded-l w-full max-w-md">
         <button type="submit" class="bg-blue-500 text-white p-2 rounded-r">Search</button>
     </form>
 
     <?php if (have_posts()) : ?>
```
Comment (andiemini7) [inline index.php]: Remove posts

File: index.php [hunk 4]
```diff
@@ -1,71 +1,95 @@
 <?php get_header(); ?>
-
 <div class="container mx-auto p-4">
```
Comment (andiemini7) [inline index.php]: Move everything with navigation from index.php to header.php

File: js/main.js [hunk 5]
```diff
@@ -0,0 +1,13 @@
+jQuery(document).ready(function($){
```
Comment (andiemini7) [inline js/main.js:1]: This needs to be added into assets/js/ create a front.js file and then another navigation.js file, add the navigation changes inside the navigation.js file and import that file inside the front.js. Fix the build from npm run build so it picks up the font.js as the js files and everything imported there

File: style.css [hunk 6]
```diff
@@ -1,6 +1,92 @@
-/*
-Theme Name: Unified-search
-Description: A blank Wordpress theme
-Theme URI: image.png
-Version: 1.0
-*/
\ No newline at end of file
+.menu-item-has-children {
```
Comment (andiemini7) [inline style.css]: Remove all this css, move to assets/scss, create a file named front.scss, then another one navigation.scss move all your css convert it to sass and then import it into font.scss then fix the npm run build pipelines to pickup the front.scss and compile it together with tailwindcss

**Review by andiemini7 — CHANGES_REQUESTED** [review]

**Review by alt1nn — COMMENTED** [review]
files moved

**Pull request merged**